x-pack/filebeat/module/threatintel/misp/ingest/pipeline.yml

```diff
@@ -241,7 +247,7 @@ processors:
            .collect(Collectors.toList());
         
         ctx.tags = tags;
-        ctx.threatintel.indicator = ['marking' : [ 'tlp': tlpTags ]];
+        ctx.threatintel.indicator.marking = [ 'tlp': tlpTags ];
```

Comment:
is it guaranteed that ctx.threatintel.indicator(.marking?) exists at this point?

Otherwise it's necessary to create `ctx.threatintel["indicator"] = new HashMap()`.

Also I'm not sure if `ctx.threatintel.indicator.marking = X` is valid when marking doesn't exists previously.

Reply:
`ctx.threatintel.indicator` was set explicitly a bit higher up.